app/views/home.njk

```diff
@@ -16,17 +16,16 @@
 
 {% block content %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-l">Payment Web</span>
-      <h1 class="govuk-heading-xl">{{serviceName}}</h1>
-
-      {{ govukButton({
-        classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8",
-        href: "/start",
-        isStartButton: true,
-        text: "Start"
-      }) }}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <h1 class="govuk-heading-l">{{ serviceName }}</h1>
 
+          <p class="govuk-body">Manage post payment tasks</p>
```

Comment:
Suggest changing this to `Manage payment services` or something.  This step is pre-payment not post payment.

Hold is just the first option we're giving users.